src/main/java/cs2103/v15_1j/jimjim/ui/TodayPaneController.java

```diff
@@ -73,17 +74,36 @@ public void refreshData(){
 		}
 
 		if(showOverdue){
-			rowFactory.showOverdue();
+			rowFactory.addLabel("Overdue", "header");
+			int noOfOverdue = rowFactory.showOverdue();
+
+			if(noOfOverdue == 0){
+				rowFactory.addLabel("Congratulations there are no Overdue Events or Tasks", "no-overdue-label");
+			}
+
+			rowFactory.addLabel("", "");
 		}
 
 		showFloatingTasks();
 	}
 
 	private void showFloatingTasks(){
+		rowFactory.addLabel("Floating Tasks", "header");
```

Comment:
Hmm, Initially I didn't put the header. But I realize that it is not as obvious, I think it looks better with the header.